Counselor's Remap

nearlyNonexistent opened the pull request:
🆑 nearlyNon
maptweak: Counselor's office reworked to have an actual desk. Much more comfortable.
/🆑

PsiOmegaDelta commented:
Just based on the Travis test results and the map bot diff you tore out all piping during the room rework.

nearlyNonexistent committed:
Counselor's Remap

🆑 nearlyNon
maptweak: Counselor's office reworked to have an actual desk. Much more comfortable.
/🆑

PsiOmegaDelta commented:
I note no major issues but is it the case that you've converted some misc. storage room into an off-the side maintenance section?

nearlyNonexistent commented:
Yes; it was done because there was no way to use all the misc storage room space, but also no way to remap counselor's office without eating some of it. All the supplies in it except the IV drip are part of the counselor's office now.
